### ISSUE TITLE
У команд spk/speak не работают параметры громкости (v) и темпа (t)

### ISSUE THREAD

**Issue by Vladislav4KZ** [FWGS/xash3d-fwgs#601]
Это также не работало в старой версии Xash3D FWGS

Я писал об этом раньше, но issue закрыли из-за того что пофиксили команду spk, но эти параметры всё ещё не пофиксили

**Comment by Vladislav4KZ** [comment]
Кстати эти аргументы также не работают в предложениях с файла sentences.txt

Например, для теста введите эту команду в GoldSource и Xash3D FWGS, и вы увидите отличия.
```
spk !HG_MONST0
```

Вот сама строка с sentences.txt
```
HG_MONST0 hgrunt/clik(p110) (t50) bogies(t0), my! sector!(t0) clik
```

В GoldSource, эта фраза проговаривается быстрее чем в ксаше, так как в ксаше не работает аргумент (t)

**Comment by a1batross** [comment]
Да, в Ксаше нет timecompress. Это известная проблема и будет скорее всего решаться на этапе переписывания микшера.

**Comment by Vladislav4KZ** [comment]
Уже неактуально. В ксаше теперь работает изменение громкости и таймкомпрессия у команды **spk**.
